resolwe_bio/kb/tests/test_mapping.py

```diff
@@ -68,7 +68,7 @@ def test_lookup(self):
         response = self.client.post(MAPPING_URL, {
             'source_db': 'SRC',
             'target_db': 'TGT',
-            'source_id': [str(x) for x in range(2048)]
+            'source_id__in': ','.join([str(x) for x in range(512)])
```

Comment:
Why have you changed the number?

Reply:
It crashes with 2048 (or 1024) so i used 512 which seems to work. :-)